docs/photoniq/fingerprinting/get-started-fingerprinting.md

```diff
@@ -0,0 +1,268 @@
+    Button: <button onclick="getVisitorDetails()">Get DS Data</button>
+</body>
+
+</html>
+```
+
+### Fingerprint on Text Input
+
+Perform a fetch call when a user starts entering some text in an input field. Here's an example:
+
+```jsx
+<html>
+<head>
+    <script>
+        // Constants
+        const DS_URL = "<URL-TO-DS-SERVICE>";
+      
+        const VISIT_API_ENDPOINT = `${DS_URL}/visits`;
+
+        let fetchPromise;
+
+        // Function to trigger fetch when someone types in the input field
+        function initiateFetch() {
+            // Check if fetchPromise is undefined to ensure fetch is called only once
+            if (!fetchPromise) {
+                window.dsPromise.then((ds) => ds.get()).then((ds_data) => {
+                    fetchPromise = fetch(VISIT_API_ENDPOINT, {
+                        method: "POST",
+                        headers: {
+                            "Content-Type": "application/json",
+                          
+                        },
+                        body: JSON.stringify(ds_data)
+                    });
+                });
+            }
+        }
+
+        // Function to handle visitor details when button is clicked
+        function getVisitorDetails() {
+            if (fetchPromise) {
+                fetchPromise
+                .then(res => {
+                    console.log("API call done!");
+                    console.log("Visitor ID from header:", res.headers.get('x-photoniq-vid'));
+                    return res.json();
+                })
+                .then(data => {
+                    console.log("Visitor ID from response:", data.visitorId);
+                    window.alert('Visitor ID from response: ' + data.visitorId);
+                })
+                .catch(e => {
+                    console.log("API call failed");
+                    console.error(e);
+                });
+            } else {
+                console.log("Data not fetched yet or there was an issue initializing the fetch.");
+            }
+        }
+
+        // Dynamically load the DS client script
+        window.onload = function () {
+            var script = document.createElement('script');
+            script.src = DS_URL;
+            script.onload = function () {
+                var dsPromise = DS_Client.load();
+                window.dsPromise = dsPromise;  // Make it globally accessible
+            };
+            document.body.appendChild(script);
+        };
+
+    </script>
+</head>
+
+<body>
+    <!-- Input field to trigger fetch when typing starts -->
+    <input type="text" oninput="initiateFetch()" placeholder="Start typing...">
+
+    <!-- Button to trigger the getVisitorDetails function -->
+    <button onclick="getVisitorDetails()">Get DS Data</button>
+</body>
+
+</html>
+```
+
+### Fingerprint with Google Tag Manager
+
+Here is an example of the JavaScript being loaded by Google’s Tag Manager. The VistorId is sent to the JavaScript console.
```

Comment:
Keep an eye on "visitorId".  Above I think I saw it with lower case, and here with upper case.  I would just standardize on the upper case one.  But if you feel different, am OK with that, as long as consistent.

Reply:
Well, which is correct and matches what we send back?

Reply:
I also prefer VisitorId, or better yet, Visitor ID, but I want to match whatever you guys did and whatever the user will see.